app/models/invitation.rb

```diff
@@ -5,6 +5,9 @@ class Invitation
   field :token, type: String
   belongs_to :event
 
+  validates_presence_of :email
+  validates_uniqueness_of :email, scope: :event
```

Comment:
Prefer the new style validations `validates :column, uniqueness: value` over `validates_uniqueness_of`.